tests/test_isrTaskLSST.py

```diff
@@ -398,10 +401,11 @@ def test_isrDark(self):
 
         delta = result2.exposure.image.array - result.exposure.image.array
         exp_time = input_exp.getInfo().getVisitInfo().getExposureTime()
+
         self.assertFloatsAlmostEqual(
             delta[good_pixels],
             self.dark.image.array[good_pixels] * exp_time,
-            atol=1e-12,
+            atol=5e-6,
```

Comment:
Do we want to change the tolerance this much, or decide which pixels are bad and not check them?  But we do expect bad CTI (trapped) pixels to be bad on this test because we are explicitly not correcting for CTI in the bootstrap pipelines.